app/policies/bgs_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+BGSPolicy = Struct.new(:user, :bgs) do
+  def access?
+    accessible = user.icn.present? && user.ssn.present? && user.participant_id.present?
+    increment_statsd_for_loa_user(accessible) if user.loa3?
+    accessible
+  end
+
+  def increment_statsd_for_loa_user(accessible)
```

Comment:
i'd drop `for_loa_user` from this method name if you are pulling out the loa3 check.
(also pull in master to stop the security warning)

Reply:
Good call